addons/website_sale/static/src/js/website_sale.js

```diff
@@ -1,4 +1,22 @@
 $(document).ready(function () {
+
+if($('ul').is('#collapse_categ')){
+    $('#collapse_categ').find('ul').addClass('collapse');
+    $('#collapse_categ').find('li.active').parents('ul').addClass('in').removeClass('collapse');
+    $('#collapse_categ').find('.active > ul').addClass('in').removeClass('collapse');
+    $('#collapse_categ').find('.parent-categ').prepend('<span class="fa fa-chevron-right"></span>');
+    $('#collapse_categ').find('ul.in').parent().find('a span')
+        .removeClass('fa-chevron-right')
+        .addClass('fa-chevron-down');
+    $('#collapse_categ').find('ul.collapse').parent().find('a span')
+        .removeClass('fa-chevron-down')
+        .addClass('fa-chevron-right');
+    $('#collapse_categ').find('.active a span').addClass('fa-chevron-down');
+    $('#collapse_categ').find('.active').children('ul').find('span')
+        .removeClass('fa-chevron-down')
+        .addClass('fa-chevron-right');
+}
```

Comment:
@jem-odoo 
I have change of behaviour of collapsible category list as per your suggesion.
Collapse category look like this : https://www.youtube.com/watch?v=BWwTIqjuj0Q&feature=youtu.be